pkgs/by-name/ra/rancid/package.nix

```diff
@@ -0,0 +1,110 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, writeShellScriptBin
+, autoreconfHook
+, libtool
+, makeBinaryWrapper
+, coreutils
+, expect
+, git
+, gnugrep
+, inetutils # for telnet
+, gnused
+, openssh
+, perl
+, runtimeShell
+}:
+
+# we cannot use resholvePackage yet - the scripts are too hairy
+
+let
+  inherit (lib) concatStringsSep makeBinPath mapAttrsToList replaceStrings;
+
+  # The installer executes ping to figure out how to call it and then sets the
+  # full path to the binary.
+  #
+  # We need the wrapped binary, so use this dumb script that pretends all is OK
+  # when run inside the builder and otherwise just calls the wrapped binary when
+  # not. This assumes that the first way the installer calls ping (with -c 1) is
+  # correct.
+  wrappedPing = writeShellScriptBin "ping" ''
+    test -x /run/wrappers/bin/ping || exit 0
+
+    exec /run/wrappers/bin/ping "$@"
+  '';
+
+  # these are binaries, not scripts
```

Comment:
Perhaps this comment could be improved?

Reply:
What exactly isn't clear? They are binaries, not scripts. ;-)